crates/sui-framework/sources/governance/genesis.move

```diff
@@ -15,7 +15,7 @@ module sui::genesis {
     const INIT_STORAGE_FUND: u64 = 1;
 
     /// Initial value of the lower-bound on the amount of stake required to become a validator.
-    const INIT_MIN_VALIDATOR_STAKE: u64 = 100000000000000;
+    const INIT_MIN_VALIDATOR_STAKE: u64 = 1;
```

Comment:
TODO: comment that this is a testnet-only value that needs to be changed might be good.

Reply:
good idea